internal/gps/manifest.go

```diff
@@ -103,7 +103,7 @@ func (m simpleRootManifest) TestDependencyConstraints() ProjectConstraints {
 func (m simpleRootManifest) Overrides() ProjectConstraints {
 	return m.ovr
 }
-func (m simpleRootManifest) IgnoredPackages() map[string]bool {
+func (m simpleRootManifest) IgnoredPackages(SolveParameters) map[string]bool {
```

Comment:
Whatever route we take to accomplishing this goal, this interface needs to remain unchanged. In particular, we don't want to muddy the waters around SolveParameters by reusing it in this way.